lib/core/client.js

```diff
@@ -422,15 +422,15 @@ class NightwatchClient extends EventEmitter {
   }
 
   get initialCapabilities() {
-    if (isObject(this.settings.capabilities)) {
+    if (isObject(this.settings.capabilities) && JSON.stringify(this.settings.capabilities) !== '{}') {
```

Comment:
This can be better written as `Object.keys(this.settings.capabilities).length > 0`